### PR TITLE
Add a README

### PR DESCRIPTION
**Pull request by ryan-blunden**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+async function main() {
+  const payload = await secrets.providers.doppler.fetch();
+  secrets.build(payload);
+}
+
+main();
+```
+
+```js
+{
```
Comment (nmanoogian) [inline README.md:33]: This is probably fairly clear already but a `// package.json` comment might be nice here

File: README.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+Take a look at the [Vercel GitOps Secrets Next.js sample repository](https://github.com/DopplerUniversity/vercel-gitops-secrets-nextjs) and deploy to Vercel to see it in action.
+
+## Contributing
+
+As this package is still in developer preview, a huge contribution you can make is simply testing this with your preferred framework and serverless provider as we'd love your feedback!
+
+You can get support in the [Doppler community forum](https://community.doppler.com/), find us on [Twitter](https://twitter.com/doppler), and for bugs or feature requests, [create an issue](https://github.com/DopplerUniversity/gitops-secrets-nodejs/issues) on this repository.
+
+We'd also love to see the number of providers grow and you can check out our [contributing guide](CONTRIBUTING.md) to get started.
```
Comment (nmanoogian) [inline README.md:176]: This link broke for me -- do we plan to add a `CONTRIBUTING.md` file in another PR?

**Review by nmanoogian — APPROVED** [review]
Looking good! Two small comments -- feel free to adjust as necessary and merge without my reapproval 👍

File: README.md (hunk 2 above)
Comment (nmanoogian) [reply README.md:176]: Disregard; I see that you added it in #5 👍

**Comment by Piccirello** [comment]
This is an excellent readme

**Pull request merged**